From 0794affd448100cf4ef8a5eacd737581a12e6a61 Mon Sep 17 00:00:00 2001
From: Min Lin <linmin@eswincomputing.com>
Date: Fri, 25 Jul 2025 11:18:52 +0800
Subject: [PATCH 636/652] fix:Enable sdio1 by default for eic7700-sbc-a1 board

Changelogs:
1.re-enable sdio1 for eic7700-sbc-a1 board, which was disabled previously

Signed-off-by: Min Lin <linmin@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dts b/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dts
index 04ffef051e20..5bba96c792d8 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dts
@@ -93,5 +93,5 @@ &d0_i2c2 {
 
 &sdio1 {
         /* disable this port */
-        status = "disabled";
+        status = "okay";
 };
\ No newline at end of file
-- 
2.53.0

